addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.plot_curve",
+    "name": "Plot Curve",
+    "description": "A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+    "version": "1.0.0",
+    "license": "Seeq",
+    "icon": "fa fa-edit",
+    "tags": {
+        "maintainer": "Seeq"
+    },
```

Comment:
Maintainer was moved out of tags, so
```
    "tags": {
        "maintainer": "Seeq"
    },
```
should just be
```
        "maintainer": "Seeq",
```

Reply:
In the next push!